### PR TITLE
Add wings42 keyboard

### PR DESCRIPTION
**Pull request by yfuku**
## Description

add wings42 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#define KC_G_TAB LGUI_T(KC_TAB)
+#define KC_G_BS LGUI_T(KC_BSPC)
+#define KC_L_SPC LT(_LOWER, KC_SPC)
+#define KC_R_ENT LT(_RAISE, KC_ENT)
+#define KC_S_JA LSFT_T(KC_LANG1)
+#define KC_S_EN LSFT_T(KC_LANG2)
+#define KC_A_DEL ALT_T(KC_DEL)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:34]:
```suggestion
    [_QWERTY] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+      KC_G_TAB, KC_A   , KC_S    , KC_D   , KC_F    , KC_G   ,     KC_H   , KC_J    , KC_K   , KC_L    , KC_SCLN, KC_G_BS ,
+    //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+       KC_LCTL, KC_Z   , KC_X    , KC_C   , KC_V    , KC_B   ,     KC_N   , KC_M    , KC_COMM, KC_DOT  , KC_SLSH, KC_RCTL,
+    //`--------+--------+---------+--------+---------+--------/   \--------+---------+--------+---------+--------+--------'
+                                  KC_A_DEL, KC_S_EN ,KC_L_SPC,    KC_R_ENT, KC_S_JA , KC_A_DEL
+    //                           `+--------+---------+--------'   `--------+---------+--------+'
+    ),
+
+    [_RAISE] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:46]:
```suggestion
    [_RAISE] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       KC_LPRN, KC_HASH, KC_DLR , KC_DQT , KC_QUOT, KC_TILD,     KC_LEFT, KC_DOWN,  KC_UP , KC_RGHT, KC_GRV , KC_RPRN,
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, _______, KC_LCBR, KC_LBRC,     KC_RBRC, KC_RCBR, _______, _______, _______, _______,
+    //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                                  _______, _______, _______,     _______, _______, RESET
+    //                          `+--------+--------+--------'   `--------+---------+--------+'
+    ),
+
+    [_LOWER] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:58]:
```suggestion
    [_LOWER] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       _______, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,     KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , _______,
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,     _______, _______, KC_COMM, KC_DOT , KC_SLSH, _______,
+    //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                                  RESET  , _______, _______,     _______, _______, _______
+    //                          `+--------+--------+--------'   `--------+--------+--------+'
+    ),
+
+    [_ADJUST] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:70]:
```suggestion
    [_ADJUST] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+#define KC_G_TAB LGUI_T(KC_TAB)
+#define KC_G_BS LGUI_T(KC_BSPC)
+#define KC_L_SPC LT(_LOWER, KC_SPC)
+#define KC_R_ENT LT(_RAISE, KC_ENT)
+#define KC_S_JA LSFT_T(KC_LANG1)
+#define KC_S_EN LSFT_T(KC_LANG2)
+#define KC_A_DEL ALT_T(KC_DEL)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/via/keymap.c:34]:
```suggestion
    [_QWERTY] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+      KC_G_TAB, KC_A   , KC_S    , KC_D   , KC_F    , KC_G   ,     KC_H   , KC_J    , KC_K   , KC_L    , KC_SCLN, KC_G_BS ,
+    //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+       KC_LCTL, KC_Z   , KC_X    , KC_C   , KC_V    , KC_B   ,     KC_N   , KC_M    , KC_COMM, KC_DOT  , KC_SLSH, KC_RCTL,
+    //`--------+--------+---------+--------+---------+--------/   \--------+---------+--------+---------+--------+--------'
+                                  KC_A_DEL, KC_S_EN ,KC_L_SPC,    KC_R_ENT, KC_S_JA , KC_A_DEL
+    //                           `+--------+---------+--------'   `--------+---------+--------+'
+    ),
+
+    [_RAISE] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/via/keymap.c:46]:
```suggestion
    [_RAISE] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       KC_LPRN, KC_HASH, KC_DLR , KC_DQT , KC_QUOT, KC_TILD,     KC_LEFT, KC_DOWN,  KC_UP , KC_RGHT, KC_GRV , KC_RPRN,
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, _______, KC_LCBR, KC_LBRC,     KC_RBRC, KC_RCBR, _______, _______, _______, _______,
+    //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                                  _______, _______, _______,     _______, _______, RESET
+    //                          `+--------+--------+--------'   `--------+---------+--------+'
+    ),
+
+    [_LOWER] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/via/keymap.c:58]:
```suggestion
    [_LOWER] = LAYOUT(
```

File: keyboards/wings42/rev1/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       _______, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,     KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , _______,
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,     _______, _______, KC_COMM, KC_DOT , KC_SLSH, _______,
+    //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                                  RESET  , _______, _______,     _______, _______, _______
+    //                          `+--------+--------+--------'   `--------+--------+--------+'
+    ),
+
+    [_ADJUST] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/via/keymap.c:70]:
```suggestion
    [_ADJUST] = LAYOUT(
```

File: keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+#define KC_G_TAB LGUI_T(KC_TAB)
+#define KC_G_BS LGUI_T(KC_BSPC)
+#define KC_L_SPC LT(_LOWER, KC_SPC)
+#define KC_R_ENT LT(_RAISE, KC_ENT)
+#define KC_S_JA LSFT_T(KC_LANG1)
+#define KC_S_EN LSFT_T(KC_LANG2)
+#define KC_A_DEL ALT_T(KC_DEL)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c:34]:
```suggestion
    [_QWERTY] = LAYOUT(
```

File: keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+---------+--------+---------+--------|                    |--------+---------+--------+---------+--------+--------|
+      KC_G_TAB, KC_A   , KC_S    , KC_D   , KC_F    , KC_G   ,   KC_DEL,  KC_DEL,   KC_H   , KC_J    , KC_K   , KC_L    , KC_SCLN, KC_G_BS ,
+    //|--------+--------+---------+--------+---------+--------|                    |--------+---------+--------+---------+--------+--------|
+       KC_LCTL, KC_Z   , KC_X    , KC_C   , KC_V    , KC_B   ,   KC_BSPC, KC_BSPC,  KC_N   , KC_M    , KC_COMM, KC_DOT  , KC_SLSH, KC_RCTL,
+    //`--------+--------+---------+--------+---------+--------/                    \--------+---------+--------+---------+--------+--------'
+                                  KC_A_DEL, KC_S_EN ,KC_L_SPC,                      KC_R_ENT, KC_S_JA , KC_A_DEL
+    //                           `+--------+---------+--------'                    `--------+---------+--------+'
+    ),
+
+    [_RAISE] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c:46]:
```suggestion
    [_RAISE] = LAYOUT(
```

File: keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|                      |--------+--------+--------+--------+--------+--------|
+       KC_LPRN, KC_HASH, KC_DLR , KC_DQT , KC_QUOT, KC_TILD,     _______, _______,  KC_LEFT, KC_DOWN,  KC_UP , KC_RGHT, KC_GRV , KC_RPRN,
+    //|--------+--------+--------+--------+--------+--------|                      |--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, _______, KC_LCBR, KC_LBRC,     _______, _______,  KC_RBRC, KC_RCBR, _______, _______, _______, _______,
+    //`--------+--------+--------+--------+--------+--------/                      \--------+--------+--------+--------+--------+--------'
+                                  _______, _______, _______,                        _______, _______, RESET
+    //                          `+--------+--------+--------'                      `--------+---------+--------+'
+    ),
+
+    [_LOWER] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c:58]:
```suggestion
    [_LOWER] = LAYOUT(
```

File: keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|                      |--------+--------+--------+--------+--------+--------|
+       _______, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,    _______, _______,   KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , _______,
+    //|--------+--------+--------+--------+--------+--------|                      |--------+--------+--------+--------+--------+--------|
+       KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,    _______, _______,   _______, _______, KC_COMM, KC_DOT , KC_SLSH, _______,
+    //`--------+--------+--------+--------+--------+--------/                      \--------+--------+--------+--------+--------+--------'
+                                  RESET  , _______, _______,                        _______, _______, _______
+    //                          `+--------+--------+--------'                      `--------+--------+--------+'
+    ),
+
+    [_ADJUST] = LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1_extkeys/keymaps/default/keymap.c:70]:
```suggestion
    [_ADJUST] = LAYOUT(
```

File: keyboards/wings42/rev1_extkeys/readme.md [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+
+![wings42](https://raw.githubusercontent.com/yfuku/wings42/main/images/wings42.jpg)
+
+A split keyboard with 3x6 vertically staggered keys and 3 thumb keys.
+
+* Keyboard Maintainer: [yfuku](https://github.com/yfuku)
+* Hardware Supported: wings42 PCB, Pro Micro
+* Hardware Availability: https://yfuku.booth.pm/
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/wings42/rev1_extkeys/readme.md:11]: Could you add instructions on how to enter the bootloader to flash new firmware?

**Review by drashna — CHANGES_REQUESTED** [review]
You have changes to a couple of git submodules.  

You'll probably need to rebase to fix the issue.

**Comment by yfuku** [comment]
> You have changes to a couple of git submodules.
> 
> You'll probably need to rebase to fix the issue.

Thank you for reviewing it.
I've fixed the git submodules.

File: keyboards/wings42/rev1_extkeys/readme.md (hunk 13 above)
Comment (yfuku) [reply keyboards/wings42/rev1_extkeys/readme.md:11]: I'm sorry. I couldn't grasp the meaning. Is it insufficient as it is?

Comment (drashna) [reply keyboards/wings42/rev1_extkeys/readme.md:11]: For instance, is there a hardware reset button?  is bootmagic enabled? And fi so, what key is the key to jump to the bootloader?   location of reset keycode in default keymap?

Comment (yfuku) [reply keyboards/wings42/rev1_extkeys/readme.md:11]: This is a keyboard with a plate for key expansion.
I'm sorry it's only in Japanese, but this is what it looks like.
https://yfuku.com/wings42/ext/keys/

Comment (yfuku) [reply keyboards/wings42/rev1_extkeys/readme.md:11]: @drashna Added the url to the build guide.

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+#define KC_G_TAB LGUI_T(KC_TAB)
+#define KC_G_BS LGUI_T(KC_BSPC)
+#define KC_L_SPC LT(_LOWER, KC_SPC)
+#define KC_R_ENT LT(_RAISE, KC_ENT)
+#define KC_S_JA LSFT_T(KC_LANG1)
+#define KC_S_EN LSFT_T(KC_LANG2)
+#define KC_A_DEL ALT_T(KC_DEL)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT(
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:34]:
```suggestion
    [_QWERTY] = LAYOUT_split_3x6_3(
```

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+      KC_G_TAB, KC_A   , KC_S    , KC_D   , KC_F    , KC_G   ,     KC_H   , KC_J    , KC_K   , KC_L    , KC_SCLN, KC_G_BS ,
+    //|--------+--------+---------+--------+---------+--------|   |--------+---------+--------+---------+--------+--------|
+       KC_LCTL, KC_Z   , KC_X    , KC_C   , KC_V    , KC_B   ,     KC_N   , KC_M    , KC_COMM, KC_DOT  , KC_SLSH, KC_RCTL,
+    //`--------+--------+---------+--------+---------+--------/   \--------+---------+--------+---------+--------+--------'
+                                  KC_A_DEL, KC_S_EN ,KC_L_SPC,    KC_R_ENT, KC_S_JA , KC_A_DEL
+    //                           `+--------+---------+--------'   `--------+---------+--------+'
+    ),
+
+    [_RAISE] = LAYOUT(
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:46]:
```suggestion
    [_RAISE] = LAYOUT_split_3x6_3(
```

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       KC_LPRN, KC_HASH, KC_DLR , KC_DQT , KC_QUOT, KC_TILD,     KC_LEFT, KC_DOWN,  KC_UP , KC_RGHT, KC_GRV , KC_RPRN,
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       _______, _______, _______, _______, KC_LCBR, KC_LBRC,     KC_RBRC, KC_RCBR, _______, _______, _______, _______,
+    //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                                  _______, _______, _______,     _______, _______, RESET
+    //                          `+--------+--------+--------'   `--------+---------+--------+'
+    ),
+
+    [_LOWER] = LAYOUT(
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:58]:
```suggestion
    [_LOWER] = LAYOUT_split_3x6_3(
```

File: keyboards/wings42/rev1/keymaps/default/keymap.c [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       _______, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,     KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , _______,
+    //|--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+       KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,     _______, _______, KC_COMM, KC_DOT , KC_SLSH, _______,
+    //`--------+--------+--------+--------+--------+--------/   \--------+--------+--------+--------+--------+--------'
+                                  RESET  , _______, _______,     _______, _______, _______
+    //                          `+--------+--------+--------'   `--------+--------+--------+'
+    ),
+
+    [_ADJUST] = LAYOUT(
```
Comment (drashna) [inline keyboards/wings42/rev1/keymaps/default/keymap.c:70]:
```suggestion
    [_ADJUST] = LAYOUT_split_3x6_3(
```

File: keyboards/wings42/rev1/rev1.h [hunk 18]
```diff
@@ -0,0 +1,44 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```
Comment (drashna) [inline keyboards/wings42/rev1/rev1.h:29]:
```suggestion
#define LAYOUT_split_3x6_3( \
```

File: keyboards/wings42/wings42.h [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+
+#pragma once
+
+#include "quantum.h"
+
+#ifdef KEYBOARD_wings42_rev1
+  #include "rev1.h"
+#endif
+#ifdef KEYBOARD_wings42_rev1_extkeys
+  #include "rev1_extkeys.h"
```
Comment (drashna) [inline keyboards/wings42/wings42.h:25]:
```suggestion
#    include "rev1.h"
#endif
#ifdef KEYBOARD_wings42_rev1_extkeys
#    include "rev1_extkeys.h"
```

File: keyboards/wings42/rev1_extkeys/rev1_extkeys.h [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+    L10, L11, L12, L13, L14, L15, E01, E03, R10, R11, R12, R13, R14, R15, \
+    L20, L21, L22, L23, L24, L25, E02, E04, R20, R21, R22, R23, R24, R25, \
+                   L30, L31, L32, R30, R31, R32 \
+    ) \
+    LAYOUT( \
+        KC_##L00, KC_##L01, KC_##L02, KC_##L03, KC_##L04, KC_##L05,                     KC_##R00, KC_##R01, KC_##R02, KC_##R03, KC_##R04, KC_##R05, \
+        KC_##L10, KC_##L11, KC_##L12, KC_##L13, KC_##L14, KC_##L15, KC_##E01, KC_##E03, KC_##R10, KC_##R11, KC_##R12, KC_##R13, KC_##R14, KC_##R15, \
+        KC_##L20, KC_##L21, KC_##L22, KC_##L23, KC_##L24, KC_##L25, KC_##E02, KC_##E04, KC_##R20, KC_##R21, KC_##R22, KC_##R23, KC_##R24, KC_##R25, \
+                                      KC_##L30, KC_##L31, KC_##L32,                     KC_##R30, KC_##R31, KC_##R32  \
+    )
```
Comment (drashna) [inline keyboards/wings42/rev1_extkeys/rev1_extkeys.h:61]: Please remove this, as the `kc` layouts cause more problems than the solve. 

```suggestion
```

File: keyboards/wings42/rev1_extkeys/readme.md (hunk 13 above)
Comment (drashna) [reply keyboards/wings42/rev1_extkeys/readme.md:11]: Thanks!

And no worries, most browsers do have translation support. Which, while isn't perfect, is usually "good enough" to understand.

**Review by drashna — CHANGES_REQUESTED** [review]
Since the layout matches the `split_3x6_3` layout, should add support for that.

would also need `LAYOUTS = split_3x6_3` in the rev1 rules.mk file.

**Comment by yfuku** [comment]
@drashna 
Thank you.
I tested it and it was fine.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**